cmd/security-proxy-setup/res/configuration.toml

```diff
@@ -54,38 +54,45 @@ RetryWaitPeriod = "1s"
   [SecretStore.Authentication]
   AuthType = "X-Vault-Token"
 
-[Clients]
-  [Clients.CoreData]
+[Routes]
+  [Routes.CoreData]
+  Name = "coredata"
```

Comment:
could you add a comment on the name is always being lower cases as internally we setup kong route that way?

Reply:
Code is still forcing it to lowercase, so case here doesn't matter.

Reply:
i think it matters. the service name is the key to the map, so the coredata != CoreData in terms of map's key.

Reply:
Lowercased `name` is used to create the proxy on line 149 & 153 of service.go

Reply:
ok, i am confused by line 254-256, which is using the name directly from config.route. so if one puts mix or big cases in the name of config file then the name to the map will be like that.
```go
	for serviceName, route := range s.configuration.Routes {
		merged[serviceName] = route
	}
```

Reply:
`serviceName` is just the map key which is not used for the route. `route` now has the Name filed which is used on lines 149 & 153 to create the actual proxy route. We no longer use the map key as part of the `proxy route`.

Reply:
i got that, it's just that we have this new "name" field from config now and it creates the contention in the map's key with it. admitted it's the same behavior as before in the map, so seeing resolved.